### ISSUE TITLE
Specify options for postcss-modules-parser

### ISSUE THREAD

**Issue by jonathanewerner** [css-modules/css-modules-require-hook#62]
I'd like to use https://github.com/gilt/postcss-less to handle less/css-modules files. This works when I modify the line 93 in `lib/index.js` as follows:

```diff
+ var lessParser = require('postcss-less').parse;

- var lazyResult = runner.process(source, assign({}, { from: filename }));
+ var lazyResult = runner.process(source, assign({}, { from: filename, parser: lessParser}));
```

It'd be nice to be able to hook into this options object from the public css-modules-require-hook API. 
What do you think?

**Comment by sullenor** [comment]
Looks good for me. Since I doubt that anyone uses the [`to` option](https://github.com/css-modules/css-modules-require-hook#to-string) I think I'll replace it with possibility to provide all options to the processor function.

**Comment by jonathanewerner** [comment]
Awesome. :)

**Comment by sullenor** [comment]
I published the new version: [4.0.0](https://github.com/css-modules/css-modules-require-hook/releases/tag/4.0.0). Check it out. Small example:

```javascript
const hook = require('css-modules-require-hook');
const lessParser = require('postcss-less').parse;

hook({
  extensions: '.less',
  processorOpts: {parser: lessParser},
});
```
